operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -1284,5 +763,516 @@ spec:
+            - patch
+            - update
+        serviceAccountName: strimzi-cluster-operator
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - "rbac.authorization.k8s.io"
+          resources:
+          # The cluster operator needs to create and manage cluster role bindings in the case of an install where a user
+          # has specified they want their cluster role bindings generated
+          - clusterrolebindings
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - delete
+          - patch
+          - update
+        - apiGroups:
+          - storage.k8s.io
+          resources:
+          # The cluster operator requires "get" permissions to view storage class details
+          # This is because only a persistent volume of a supported storage class type can be resized
+          - storageclasses
+          verbs:
+          - get
+        - apiGroups:
+          - ""
+          resources:
+          # The cluster operator requires "list" permissions to view all nodes in a cluster
+          # The listing is used to determine the node addresses when NodePort access is configured
+          # These addresses are then exposed in the custom resource states
+          # The Kafka Brokers require "get" permissions to view the node they are on
+          # This information is used to generate a Rack ID that is used for High Availability configurations
+          - nodes
+          verbs:
+          - get
+          - list
+        serviceAccountName: strimzi-cluster-operator
+      deployments:
+      - name: amq-streams-cluster-operator-v1.7.0
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: strimzi-cluster-operator
+              strimzi.io/kind: cluster-operator
+          template:
+            metadata:
+              labels:
+                name: strimzi-cluster-operator
+                strimzi.io/kind: cluster-operator
+            spec:
+              serviceAccountName: strimzi-cluster-operator
+              volumes:
+                - name: strimzi-tmp
+                  emptyDir:
+                    medium: Memory
+                - name: co-config-volume
+                  configMap:
+                    name: strimzi-cluster-operator
+              containers:
+                - name: strimzi-cluster-operator
+                  image: registry.redhat.io/amq7/amq-streams-rhel7-operator@sha256:0e37f671e6d489c3e56ffb8f51f19a2a014b2cb3f57eafcc5e2931090932e725
+                  ports:
+                    - containerPort: 8080
+                      name: http
+                  args:
+                    - /opt/strimzi/bin/cluster_operator_run.sh
+                  volumeMounts:
+                    - name: strimzi-tmp
+                      mountPath: /tmp
+                    - name: co-config-volume
+                      mountPath: /opt/strimzi/custom-config/
+                  env:
```

Comment:
You lost the 3scale annotations and labels.

Reply:
Added them back in